webapp/src/routes/EmailVerification/EmailVerification.js

```diff
@@ -6,42 +6,82 @@ import CircularProgress from '@material-ui/core/CircularProgress'
 import Button from '@material-ui/core/Button'
 import Grid from '@material-ui/core/Grid'
 import Typography from '@material-ui/core/Typography'
+import { Box } from '@material-ui/core'
 import { useTranslation } from 'react-i18next'
+import { useHistory } from 'react-router-dom'
 
+import { useUser } from '../../context/user.context'
 import CustomRouterLink from '../../components/CustomRouterLink'
 import { VERIFY_EMAIL } from '../../gql'
 
 const useStyles = makeStyles((theme) => ({
   root: {
     padding: theme.spacing(4),
     display: 'flex',
-    height: '100vh',
-    width: '100vw',
-    backgroundColor: '#ffffff'
+    height: 'calc(100vh - 60px)'
   },
   content: {
     justifyContent: 'center',
     alignItems: 'center',
     display: 'flex'
   },
   centerText: {
-    textAlign: 'center'
+    textAlign: 'center',
+  },
+  tittle: {
+    fontFamily: "Roboto",
```

Comment:
Roboto Font is now preloaded. So, it's not necessary to specify Roboto in the css styles